prometheus_client/exposition.py

```diff
@@ -135,15 +135,16 @@ def start_wsgi_server(port, addr='', registry=REGISTRY):
 start_http_server = start_wsgi_server
 
 
-def generate_latest(registry=REGISTRY):
+def generate_latest(registry=REGISTRY, extra_labels={}):
```

Comment:
If this is to be accepted we would also want this in `openmetrics/exposition.py`.